.pre-commit-config.yaml

```diff
@@ -1,6 +1,22 @@
 repos:
--   repo: https://github.com/ambv/black
+  - repo: https://github.com/ambv/black

```

Comment:
@jgrss ,I am not too familiar with this file set up so @chipindigo probably knows more.
I was just looking at an example from https://github.com/pre-commit/pre-commit/blob/main/.pre-commit-config.yaml and their indentation is slightly different.